Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -35,6 +36,23 @@ class ShellException(Exception):
 IMG_FORMATS = ['jpg', 'jpeg', 'png', 'gif']
 
 
+def handle_error_read_only(fun, path, exp):
```

Comment:
Since the function isn't expected to be called by the script's code, it's best to add a docstring explaining this (e.g. `handle errors that can be encountered in shutil.rmtree() execution`).